cluster/addons/e2e-rbac-bindings/random-addon-grabbag.yaml

```diff
@@ -1,20 +0,0 @@
-# TODO remove this
-# currently, the kube-addon-manager is adding lots of pods which all share
-# the system:serviceaccount:kube-system:default identity.  We need to subdivide
-# those service accounts, figure out which ones we're going to make bootstrap roles for
-# and bind those particular roles in the addon yaml itself.  This just gets us started
-apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
We could create a binding that does nothing, and set it as an EnsureExists addon. New clusters would get the toothless policy created, but existing clusters would see that something by that name already exists. It does mean that we have a binding called "todo-remove-grabbag-cluster-admin" forever...